config.gradle

```diff
@@ -16,7 +16,7 @@ ext {
     ]
 
     libraries = [
-            godotAndroidLib: "org.godotengine:godot:4.4.0.beta-SNAPSHOT",
+            godotAndroidLib: "org.godotengine:godot:4.4.1.stable",
```

Comment:
Ooo, good catch! Theoretically, this could be split from this PR, but I think this is fine